components/datetime/src/time_zone.rs

```diff
@@ -44,6 +43,7 @@ pub(crate) enum ResolvedNeoTimeZoneSkeleton {
     IsoXXXXX,
     // TODO:
     // `VV` "America/Los_Angeles"
+    // `VVV` "Los Angeles"
     // Generic Partial Location: "Pacific Time (Los Angeles)"
```

Comment:
nit: remove this, it's implemented

actually, do we need the full exemplar city names for it?

Reply:
hmm, that's annoying. Maybe not though because it seems Generic Partial Location is only used when you're not the primary zone?

Reply:
I think it's fine, because #5657 uses the new logic for the bracketed part, i.e. it will say `Central European Time (Germany)` and `Pacific Time (Los Angeles)`.

Reply:
It would actually be nicer to say `Pacific Time (United States)`, but I'd have to check if the metazone invariants allow that.

Reply:
`Pacific Time (United States)` sounds extremely weird.

Reply:
Ah, `Mountain Time (United States)` doesn't work.

Reply:
> Pacific Time (United States) sounds extremely weird.

Weirder than `Pacific Time (Los Angeles)`? It's the Pacific as used in the United States, following United States DST rules. They're not Los Angeles DST rules. Does `Pacific Time (Mexico)` sound weird to you as well?

Reply:
I think we're safe there because we only do the region override if the time zone is the only one in the country, so the city and region are interchangeable. Right?

Reply:
> I think we're safe there because we only do the region override if the time zone is the only one in the country, so the city and region are interchangeable. Right?

Yes, current behaviour is safe. The proposed behaviour of `Pacific Time (Canada)` needs more work, because we need to know whether a metazone+country combination is unique. `ampa`+`US` is, `ammo`+`US` isn't.